woocommerce/woocommerce-ios
# Add screenshot mocks

Add support for screenshot mocks.

**To Test:**
- Ensure that all screenshots CI tasks pass

There's a _lot_ of code here – the majority of it is the mocks themselves, which aren't particularly interesting.

@shiki – this follows the pattern that you laid out previously, so I'm not sure if you want to take another look at it from an architectural standpoint? I think we'd talked about punting this all into a separate Swift package, which I'm ok to do if you'd like to, but I wasn't sure if it'd still apply given the move to Yosemite?

@rachelmcr and @JavonDavis  – you've previously worked on screenshots, so tagging you though you're both optional :) 

@woocommerce/release-managers – hoping at least one of you can take a detailed look through this (sorry!!)
